gap/examples.gi

```diff
@@ -714,3 +714,41 @@ function(filt, m, n)
   SetIsSymmetricDigraph(D, true);
   return D;
 end);
+
+InstallMethod(BookDigraphCons,
+  "for IsMutableDigraph and one positive integer",
+  [IsMutableDigraph, IsPosInt],
+  function(filt, m)
+    local pages, book, graph;
+    # special cases
+    if m = 1 then
+      # "the" square graph
+      return CompleteBipartiteDigraph(filt, 2, 2);
+    elif m = 2 then
+      # domino graph
+      return SquareGridGraph(filt, 2, 3);
+    fi;
+    pages := StarDigraph(filt, m);
+    book := ChainDigraph(filt, 2);
+    graph := DigraphCartesianProduct(book, pages);
+    return graph;
```

Comment:
Overall, I think the method could be fixed and simplified as follows:
```suggestion
    local book;
    book := CompleteDigraph(IsMutableDigraph, 2);
    return DigraphCartesianProduct(book, StarDigraph(IsMutableDigraph, m + 1));
```

Reply:
a lot simpler 😅, thank you for this.